### PR TITLE
Adding data connection info to workbench config steps

### PR DESCRIPTION
**Pull request by syaseen-rh**
## Description
As per https://issues.redhat.com/browse/RHODS-12365, steps to add a data connection needed to be added to the workbench creation section: [Chapter 5. Creating a project workbench](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science_self-managed/1.33/html-single/getting_started_with_red_hat_openshift_data_science_self-managed/index#creating-a-project-workbench_get-started)

I've added the steps as required..

## How Has This Been Tested?
Validated with a local build of the Open Data Hub website. Preview of content attached below.
Screenshot 2023-11-16 at 4 49 10 PM

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: modules/creating-a-project-workbench.adoc [hunk 1]
```diff
@@ -55,6 +55,23 @@ endif::[]
 .. Configure the storage for your {productname-short} cluster.
 ... Select *Create new persistent storage* to create storage that is retained after you log out of {productname-short}. Complete the relevant fields to define the storage.
 ... Select *Use existing persistent storage* to reuse existing storage and select the storage from the *Persistent storage* list.
+.. To use a *Data connection*, select the *Use a data connection* check box.
```
Comment (jbyrne-redhat) [inline modules/creating-a-project-workbench.adoc:58]: Bolding a noun (in this case **Data connection**) to also represent the name of a label or section in the UI goes against our style guidelines. Also, IBM style advises "checkbox" (see https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#checkbox).

Therefore, I suggest something like the following: 

`To use a data connection, in the *Data connections* section, select the *Use a data connection* checkbox.`

(I know that there's repetition in that, but I think it's the appropriate formulation).

File: modules/creating-a-project-workbench.adoc [hunk 2]
```diff
@@ -55,6 +55,23 @@ endif::[]
...
++
+--
+* To create a *new data connection*
```
Comment (jbyrne-redhat) [inline modules/creating-a-project-workbench.adoc:61]: Suggest: `To create a new data connection, perform the following actions:`

File: modules/creating-a-project-workbench.adoc [hunk 3]
```diff
@@ -55,6 +55,23 @@ endif::[]
...
+... Select *Create new data connection*.
+... In the *Name* field, enter a unique name for the data connection.
+... In the *Access key* field, enter your access key ID for Amazon Web Services (AWS).
+... In the *Secret key* field, enter your secret access key for the AWS account you specified.
+... In the *Endpoint* field, enter the endpoint of your AWS S3 storage.
+... In the *Region* field, enter the default region of your AWS account.
+... In the *Bucket* field, enter the name of the AWS S3 bucket.
+
+* To use an *existing data connection*
```
Comment (jbyrne-redhat) [inline modules/creating-a-project-workbench.adoc:70]: Similarly, I suggest: `Use an existing data connection as follows:`

File: modules/creating-a-project-workbench.adoc [hunk 4]
```diff
@@ -55,6 +55,23 @@ endif::[]
...
 ... Select *Create new persistent storage* to create storage that is retained after you log out of {productname-short}. Complete the relevant fields to define the storage.
 ... Select *Use existing persistent storage* to reuse existing storage and select the storage from the *Persistent storage* list.
+.. To use a *Data connection*, select the *Use a data connection* check box.
++
+--
+* To create a *new data connection*
+... Select *Create new data connection*.
+... In the *Name* field, enter a unique name for the data connection.
+... In the *Access key* field, enter your access key ID for Amazon Web Services (AWS).
+... In the *Secret key* field, enter your secret access key for the AWS account you specified.
```
Comment (jbyrne-redhat) [inline modules/creating-a-project-workbench.adoc:65]: Suggest: "....that you specified."

**Review by jbyrne-redhat — CHANGES_REQUESTED** [review]
This looks good, @syaseen-rh. Just a few small updates to suggest.

File: modules/creating-a-project-workbench.adoc (hunk 1 above)
Comment (syaseen-rh) [reply modules/creating-a-project-workbench.adoc]: Thanks @jbyrne-redhat - I just addressed all the comments in the latest commit, please review.

Comment (syaseen-rh) [reply modules/creating-a-project-workbench.adoc]: Screenshot 2023-11-17 at 9 45 27 AM

File: modules/creating-a-project-workbench.adoc [hunk 5]
```diff
@@ -55,6 +55,23 @@ endif::[]
 .. Configure the storage for your {productname-short} cluster.
 ... Select *Create new persistent storage* to create storage that is retained after you log out of {productname-short}. Complete the relevant fields to define the storage.
 ... Select *Use existing persistent storage* to reuse existing storage and select the storage from the *Persistent storage* list.
+.. To use a data connection, in the *Data connections* section, select the *Use a data connection* checkbox.
++
+--
+* To create a new data connection, perform the following actions:
```
Comment (jbyrne-redhat) [inline modules/creating-a-project-workbench.adoc:61]: Just one remaining update here, @syaseen-rh. See @https://github.com/opendatahub-io/opendatahub-documentation/pull/76#discussion_r1397378293.

I changed my original suggestion, so blame me! :)

File: modules/creating-a-project-workbench.adoc (hunk 2 above)
Comment (syaseen-rh) [reply modules/creating-a-project-workbench.adoc]: Updated the line as suggested:
Screenshot 2023-11-17 at 10 35 52 AM

File: modules/creating-a-project-workbench.adoc (hunk 5 above)
Comment (syaseen-rh) [reply modules/creating-a-project-workbench.adoc]: @jbyrne-redhat Addressed: 
Screenshot 2023-11-17 at 10 35 52 AM

**Review by jbyrne-redhat — APPROVED** [review]

**Pull request merged**